### PR TITLE
COTECH-681 Quick fix for wrong reporting data in JWP

### PR DESCRIPTION
**Pull request by nandy-andy**

### CODE REVIEW

File: stand-alone/yarn.lock [hunk 1]
```diff
@@ -4697,6 +4692,22 @@ styled-components@^5.0.1:
     shallowequal "^1.1.0"
     supports-color "^5.5.0"
 
+styled-components@^5.3.3:
```
Comment (kwfandom) [inline stand-alone/yarn.lock:4695]: im a newbie, but i thought that we've deleted styled components

Comment (andrzejkupczyk) [reply stand-alone/yarn.lock:4695]: Nice observation. The `package.json` file remains unchanged, so I'm curious about the alterations in the lock file. 🤔

**Review by andrzejkupczyk — APPROVED** [review]

File: stand-alone/yarn.lock (hunk 1 above)
Comment (nandy-andy) [reply stand-alone/yarn.lock:4695]: Damn, I might've push it accidentally. Thanks guys for catching! Let me look into it...

File: src/jwplayer/players/RedVentureVideoPlayer/redVenturePlayerOnReady.ts [hunk 2]
```diff
@@ -2,13 +2,10 @@ import { communicationService } from 'jwplayer/utils/communication';
 import { recordVideoEvent, VIDEO_RECORD_EVENTS } from 'jwplayer/utils/videoTimingEvents';
 import { RedVentureVideoDetails } from 'jwplayer/types';
 
-let jwPlayerKeyCounter = 0;
-
 export default function useOnRedVenturePlayerReady(videoDetails: RedVentureVideoDetails, playerInstance): void {
 	// Increment the jwPlayerKeyCounter in cases where multiple video players have to be embedded.
```
Comment (gaperlinski) [inline src/jwplayer/players/RedVentureVideoPlayer/redVenturePlayerOnReady.ts:6]: is this comment still needed?

File: src/jwplayer/players/RedVentureVideoPlayer/redVenturePlayerOnReady.ts [hunk 3]
```diff
@@ -2,13 +2,10 @@ import { communicationService } from 'jwplayer/utils/communication';
...
 	// A unique playerKey will allow control of each unique JW Player that's embedded on the page, through the JW Player API.
-	const playerKey = 'aeJWPlayerKey' + (jwPlayerKeyCounter === 0 ? '' : jwPlayerKeyCounter);
```
Comment (gaperlinski) [inline src/jwplayer/players/RedVentureVideoPlayer/redVenturePlayerOnReady.ts:10]: how is `playerInstance.id` set?

File: src/jwplayer/utils/globalJWInterface.ts [hunk 4]
```diff
@@ -6,97 +6,97 @@
 import { Player } from 'jwplayer/types';
 
 export interface WindowWithJWPlayer {
-	jwplayer: () => Player;
+	jwplayer: (playerId: string | null) => Player;
```
Comment (gaperlinski) [inline src/jwplayer/utils/globalJWInterface.ts:9]: is the implementation of this function somewhere else?

**Comment by gaperlinski** [comment]
Once this is merged, how do we bump the version in Metacritic neutorn?

File: src/jwplayer/utils/globalJWInterface.ts (hunk 4 above)
Comment (nandy-andy) [reply src/jwplayer/utils/globalJWInterface.ts:9]: Yes, it's an external code. Created by 3rd-party JWPlayer: https://docs.jwplayer.com/players/reference/javascript-player-api-introduction

**Comment by nandy-andy** [comment]
Once this is merge we manually release the new version to GCS following the instructions to https://github.com/Wikia/jwplayer-fandom/blob/master/stand-alone/README.md. Once it's released to GCS it's done as Goliath and other N&R sites load `https://static.wikia.nocookie.net/silversurfer/video/prod/standalone-dist/standAlone_RV_VideoPlayer.js` for example: 
* https://github.com/Wikia/neutron-tvguide/blob/a4e1f4fa1989b5674c6732a2a672c00f20364a46/src/resources/js/mixins/head/values/scripts.ts#L84-L92
* https://github.com/Wikia/gamefaqs/blob/22d9da58e18ed7121c8460351b6ecafb0e801ec5/templates/gamespace/gamespot_embed.tpl#L2
* https://github.com/Wikia/metacritic/blob/f14dcc72e68150e48b155254fac1eec26ba77f19/templates/WWW/layouts/header/base.tpl#L155
* https://github.com/Wikia/phoenix/blob/dedc8e87ad0b7cabe8a6f1095c512bb588f3db1a/SiteBundle/Resources/views/Core/base.html.twig#L237-L239

I'm not sure how Goliath loads the asset, though. Maybe GTM? 🤔

**Comment by andrzejkupczyk** [comment]
@nandy-andy, the script is loaded via UnifiedHeadMixin decorator.

https://github.com/Wikia/neutron-metacritic/blob/mc-goliath/src/resources/js/mixins/head/values/scripts.ts#L110

**Pull request merged**